test-common/build.gradle

```diff
@@ -1,6 +1,7 @@
 description = 'Test utility library, generic test utilities useful for TripleA projects'
 
 dependencies {
+    implementation "commons-io:commons-io:$commonsIoVersion"
```

Comment:
I already mentioned it, I don't think it's worth using this dependency, test-common could just implement helper methods in 2 or 3 lines and we have the same functionality without any additional dependencies

Reply:
No objection, thanks for the suggestions
Updated: 685cda865c5fd9a4efb1187843dcccf29ab6a59b